src/app/package-json.js

```diff
@@ -86,6 +86,7 @@ const packageJSON = (current, context) => {
     'version': context.version,
     'description': context.description,
     'main': 'es5/plugin.js',
+    'browser': 'src/plugin.js',
```

Comment:
since we don't configure browserify to run the `babelify` transform, we actually don't want to do this. Browserify would then try to pick up the src and that's es6 code and it'll fail.